script.js

```diff
@@ -1,4 +1,195 @@
+var WinCounter = 0;
+var LoseCounter = 0;
+var DrawCounter = 0;
+var TotalTurns = 0;
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  var myArray = ["scissors", "paper", "stone"];
+  var ComputerChoice = myArray[Math.floor(Math.random() * myArray.length)];
```

Comment:
Neat way of doing this. But might want to make this a function so that its easy to reuse